flashinfer/autotuner/autotuner.py

```
@@ -1059,8 +1191,8 @@ def _apply_tuning_overrides(self, tuning_config: TuningConfig) -> TuningConfig:
                     gen_fn = spec.gen_tuning_buckets
                     new_gen = gen_fn
 
-                    def _clamped_po2_mapper(x, _gen_fn=gen_fn):
-                        buckets = tuple(sorted(set(_gen_fn(x))))
+                    def _clamped_po2_mapper(x: int):
+                        buckets = tuple(sorted(set(gen_fn(x))))
```

Ruff (0.15.18)

[warning] Function definition does not bind loop variable `gen_fn`

(B023)